cgo/lapack_test.go

```diff
@@ -156,7 +156,7 @@ func TestDorghr(t *testing.T) {
 }
 
 func TestDormqr(t *testing.T) {
-	testlapack.Dorm2rTest(t, blockedTranslate{impl})
+	testlapack.DormqrTest(t, blockedTranslate{impl})
```

Comment:
This wasn't a typo. The test for Dormqr just compares to the output of Dorm2r. The cgo dorm2r is just a call to Dormqr, so this change just confirms that Dormqr = Dormqr. We want to call Dorm2rTest to actually test that the routine computes a QR factorization.